### DISCUSSION TITLE
Gosund SmartPlug Power Meter

### DISCUSSION THREAD

**Discussion by Stevek83** [milo526/homebridge-tuya-web#238] [Q&A]
Hi,

is there any chance to show values from the power meter in homekit?

Thx for your help

Stefan

**Comment by milo526** [comment]
No sadly not. The api does not support this use-case.

**Reply by Stevek83** [reply]
Hi Milo,

thx for your fast answer. Is there any chance to support this in the future?
In my SmartLife-App I can see this values so it should be possible to read the values from the device.

Stefan

**Reply by milo526** [reply]
No, it is not possible.

Tuya and SmartLife apps use different apis then this plugin.
You can look for yourself, this information is not present in the data that is received. Many have looked for it before but not found it.

**Reply by Stevek83** [reply]
Hi Milo,

thx for this information so I don't need to look for it anymore.

Nevertheless you made a nice plugin thx for that.

Stefan
